SwiftWisdom/Core/Geometry/Geometry.swift

```diff
@@ -86,6 +86,11 @@ public func random(range: Range<Int>) -> Int {
 }
 
 public func avg(numbers: CGFloat...) -> CGFloat {
+    return avg(numbers)
+}
+
+public func avg(numbers: [CGFloat]) -> CGFloat {
```

Comment:
I prefer not to deal w/ optionals in collections and numbers for the most part.  If the array is empty and shouldn't average, the user can check it before running average. @cprime thoughts?